### DISCUSSION TITLE
Clown Role

### DISCUSSION THREAD

**Discussion by ReitoMakara** [FakeFishGames/Barotrauma#7623] [Ideas and suggestions]
**An assistant's talent tree currently forces them to become a clown at some point and respawn without a clown suit is a bit annoying so a clown-specific role would be welcome.**

In my concept he would function as a melee security guard and dysfunctional in other aspects (For funny reasons) like fixing things in a conventional way (Using a screwdriver or a wrench), using the submarine's weapons and piloting the submarine. This dysfunction would serve to discern the learning curve of a clown and an assistant as both would have no specific responsibilities or high skills.

As you understand in the "Clown" tree, the usefulness of this position would be based on buffing the team members and being resistant. So a few item ideas would be things like: 
- Attractive horn - Invading monsters will focus in the horn honk and ignore the others crew members.
- Balloons - When popped stun or scare monsters.
- Banana peels - Makes those who pass for it to fall and make a funny noise.
These would be useful and funny mechanics.

For the assistants benefits it could get a new talent tree so this can also brings more utility for both.
- Mine idea would be a tree based in the first talent that he reaches a certain level. For exemple: If he gets level 50 Mechanical unlocks Technician talent tree.

**Comment by ghost** [comment]
1. Assistant is never forced to be a clown, in fact I typically take the clown tree last. 
2. Death should be bad. Being able to die and have virtually no losses is a terrible suggestion.
3. If you want stupid clown roles and bananas, [There's a mod for that.](https://steamcommunity.com/sharedfiles/filedetails/?id=2660379561&searchtext=thalassophobia)
